snappiershot/snapshot/snapshot.py

```diff
@@ -29,7 +29,9 @@ def __init__(self, configuration: Optional[Config] = None) -> None:
         self._metadata: Optional[SnapshotMetadata] = None
         self._snapshot_file: Optional[_SnapshotFile] = None
 
-    def assert_match(self, value: Any, exact: bool = False, update: bool = False) -> bool:
+    def assert_match(
+        self, value: Any, exact: bool = False, update: bool = False, ignore: list = []
```

Comment:
I think `ignore` must be a list of `str`, if I am reading this all correctly. Should be typed as such throughout

Reply:
[fixed in this commit](https://github.com/MORSECorp/snappiershot/pull/96/commits/d2b4e83bb111cc47cb4d2b52c2940770492218d8)